.pre-commit-config.yaml

```diff
@@ -31,8 +31,3 @@ repos:
     - id: pretty-format-json
       args: [--autofix]
     - id: trailing-whitespace
-- repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
-  rev: v2.1.0
-  hooks:
-    - id: pretty-format-java
-      args: [--autofix]
```

Comment:
Bunu geri ekleyelim @emrecandabak #10'da söylediğim üzere başka bir PR açacak, bunun için